### ISSUE TITLE
Enable UARTs 1, 4 and 5 on Beaglebone Black

### ISSUE THREAD

**Issue by GuruprashanthKrishnakumar** [cu-ecen-aeld/final-project-jordicrosmompart#3]
**Definition of Done (DoD)**

- Create the Device Tree patch that will be used by the kernel image

**Blockers**

- [Bring up of Buildroot environment](https://github.com/cu-ecen-aeld/final-project-jordicrosmompart/issues/1) - COMPLETED

**Comment by jordicrosmompart** [comment]
Update: The latest device tree is not compatible with the one provided in (https://github.com/cu-ecen-aeld/buildroot-assignments-base/blob/hw-beaglebone-black-uart/base_external/patches/linux/0001-Added-UART-1-4-and-5-support-for-BeagleBone-Black.patch). 
We need to find a way to enable UART

**Comment by GuruprashanthKrishnakumar** [comment]
Closed: Created a new patch that updated am335x-bone-common.dtsi as per the latest beagleboard linux repo. Updated am33xx-l4.dtsi to enable the UARTs. UARTs are still not enabled. 

We cloned Mohit's [repo](https://github.com/MohitRane8/buildroot-assignments-base/tree/hw-beaglebone-black-uart) which was when the patch was deployed and built the kernel image with same configurations and were able to enable UARTs:
![image](https://user-images.githubusercontent.com/97482748/201591124-73bf9165-5cf5-4d56-9140-03fec97689bc.png)

A follow-up [issue](https://github.com/cu-ecen-aeld/final-project-jordicrosmompart/issues/12) to make this patch work on a later version of buildroot (if not the latest version) .
